src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [unlCount, setUnlCount] = useState(0)
+  const [stableVersion, setStableVersion] = useState<string | null>(null)
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  const aggregateData = (v: any[]) => {
+    if (!v) {
+      return []
+    }
+    let total = 0
+    const tempData: any[] = []
+    v.reduce((res, val) => {
+      if (!res[val.server_version]) {
+        res[val.server_version] = {
+          server_version: val.server_version,
+          count: 0,
+        }
+        tempData.push(res[val.server_version])
+      }
+      res[val.server_version].count += 1
+      total += 1
+      return res
+    }, {})
+
+    return tempData
+      .map((item) => ({
+        label: item.server_version ? item.server_version : ' NA ',
```

Comment:
nit: I think the `/` is usually included for the Not Applicable acronym
```suggestion
        label: item.server_version ? item.server_version : ' N/A ',
```